### PR TITLE
Add MeshModel endpoints to swagger docs

### PR DESCRIPTION
**Pull request by Revolyssup**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 06f3f35be08f457df40cb820936a27351db981cd at: https://63ac8e9660b03a1f994f6d5b--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b051e375f96c088c27819b72276708eca22483ca at: https://63ac93894c91171f04b6ee38--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 08c9bed16ce9f6a6071416ccc59fe0388a05c71c at: https://63ac95063c8e6c2215944264--meshery-docs-preview.netlify.app

File: server/models/pattern/core/pattern.go [hunk 1]
```diff
@@ -388,7 +388,6 @@ func NewPatternFileFromK8sManifest(data string, ignoreErrors bool) (Pattern, err
 }
 
 func createPatternServiceFromK8s(manifest map[string]interface{}) (string, Service, error) {
-
```
Comment (leecalcote) [inline server/models/pattern/core/pattern.go:391]: gofmt. Thanks for this.

File: server/handlers/component_handler.go [hunk 2]
```diff
@@ -154,6 +177,14 @@ func (h *Handler) MeshmodelComponentTypesHandler(rw http.ResponseWriter, r *http
 		http.Error(rw, ErrWorkloadDefinition(err).Error(), http.StatusInternalServerError)
 	}
 }
+
+// swagger:route GET /api/meshmodel/components/{type} MeshmodelGetByType idMeshmodelGetByType
+// Handle GET request for getting meshmodel components of a specific type.
+// Example: /api/meshmodel/components/kubernetes
```
Comment (leecalcote) [inline server/handlers/component_handler.go:183]: Must be lowercase? 

Put the answer into the code comment, if you would.

**Comment by leecalcote** [comment]
updating labels

**Comment by l5io** [comment]
🚀 Preview for commit 1c77895fc761ef4a73008813273a9fc59cec13b6 at: https://63ad72c76c05821028e9c6a2--meshery-docs-preview.netlify.app

File: server/handlers/component_handler.go [hunk 3]
```diff
@@ -98,6 +104,14 @@ func (h *Handler) GetAllMeshmodelComponents(rw http.ResponseWriter, r *http.Requ
 		http.Error(rw, ErrWorkloadDefinition(err).Error(), http.StatusInternalServerError)
 	}
 }
+
+// swagger:route GET /api/meshmodel/components/{type}/{name} MeshmodelGetByName idMeshmodelGetByName
+// Handle GET request for getting meshmodel components of a specific type by name.
+// Example: /api/meshmodel/components/kubernetes/Namespace
+// Components can be further filtered through query parameter ?version=
```
Comment (leecalcote) [inline server/handlers/component_handler.go:111]: Would it be ok if we moved to `ver`?

**Review by leecalcote — COMMENTED** [review]
I'll let you track the `ver` request in the API endpoints spreadsheet.

**Comment by leecalcote** [comment]
Merge conflict.

**Pull request merged**